op-deployer/pkg/deployer/artifacts/locator.go

```diff
@@ -133,5 +124,13 @@ func unmarshalURL(text string) (*Locator, error) {
 		return nil, err
 	}
 
-	return &Locator{URL: u}, nil
+	tag := u.Fragment
+	if tag == "" {
+		tag = standard.DevTag
+	}
+	if !standard.IsKnownTag(tag) {
+		return nil, fmt.Errorf("unknown tag: %s", tag)
+	}
```

Comment:
Thinking about this more. It seems like users can only use the url fragment with one of the following 4 values:
```
	ContractsV160Tag        = "op-contracts/v1.6.0"
	ContractsV180Tag        = "op-contracts/v1.8.0-rc.4"
	ContractsV170Beta1L2Tag = "op-contracts/v1.7.0-beta.1+l2-contracts"
	DevTag                  = "op-contracts/dev"
```
Can you provide a specific example of how the url fragment would be used? You just want to allow downloading one of the standard contracts package from an unknown server, but still verify the content hash matches expectation?